jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/ImageName.java

```diff
@@ -288,6 +294,11 @@ private void doValidate() {
                         checks[i], value, checkPattern.pattern()));
             }
         }
+
+        if (repository.length() > REPO_NAME_MAX_LENGTH) {
+            errors.add(String.format("Repository name must not be more than %d characters", REPO_NAME_MAX_LENGTH));
```

Comment:
Could you please add one more test in ImageNameTest for verifying exception with this exact message is thrown when too long image name is provided?